custom-login/src/Login.jsx

```diff
@@ -18,8 +18,13 @@ import config from './config';
 
 const Login = () => {
   const { oktaAuth } = useOktaAuth();
+  const widgetRef = useRef();
 
   useEffect(() => {
+    if (!widgetRef.current) {
```

Comment:
You probably want to use `useLayoutEffect` here and then you don't need this check.

https://reactjs.org/docs/hooks-reference.html#uselayouteffect